bool subscribe
uint16 freqAcq
uint32 bufferSize
---
int8 ret